test_redirects/map_globalconf.py

```diff
@@ -938,5 +938,8 @@
 
     # bug 1238248
     url_test('/firefox/push/',
-             'https://support.mozilla.org/kb/push-notifications-firefox')
+             'https://support.mozilla.org/kb/push-notifications-firefox'),
+
+    # bug 1237875
+    url_test('/community/forums/', '/about/forums/')
```

Comment:
NIT: Always best to add the comma at the end for when we need to add the next one. Not a requirement, just FYI.